test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -4,6 +4,7 @@
 import * as chai from 'chai';
 import * as sinon from 'sinon';
 
+import { Device, VideoTransformDevice } from '../../src';
```

Comment:
These, ironically, we prefer to import from the individual source files.

Reply:
Fixed.